# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1458249149 25200
#      Thu Mar 17 14:12:29 2016 -0700
# Node ID 6e5540ddb6392a746bfc04035908a73026740fad
# Parent  a4c411681e256ce3071a2039b6f18597018f12ba
textscan: Don't #include builtin-defun-decls.h

* textscan.cc: Don't #include "builtin-defun-decls.h", it is not needed
and causes a circular dependency in the build system.

diff --git a/libinterp/corefcn/textscan.cc b/libinterp/corefcn/textscan.cc
--- a/libinterp/corefcn/textscan.cc
+++ b/libinterp/corefcn/textscan.cc
@@ -1053,17 +1053,16 @@ textscan::scan_one (dstr& is, const text
                                               Cell (octave_value (vv)));
         // FIXME -- why does failbit get set at EOF, instead of eofbit?
       if (vv.length () != 0)
         is.clear (is.rdstate () & ~std::ios_base::failbit);
     }
   is.field_done ();
 }
 
-#include "builtin-defun-decls.h"
 /**
  *  Read data corresponding to the entire format string once,
  *  placing the values in row  row of retval.
  */
 int
 textscan::read_format_once (dstr& is,
                             textscan_format_list& fmt_list,
                             std::list<octave_value> & retval,
